flowey/flowey_lib_common/src/lib.rs

```diff
@@ -32,8 +32,8 @@ pub mod download_protoc;
 pub mod gh_download_azure_key_vault_secret;
 pub mod gh_task_azure_login;
 pub mod git_checkout;
-pub mod install_apt_pkg;
```

Comment:
you deleted the mod declaration, but your PR doesn't actually delete `install_apt_pkg.rs`. pls fix :)

Reply:
Done, thanks!